### PR TITLE
fix: transcriber settings precedence

### PR DESCRIPTION
**Pull request by rpurdel**

### CODE REVIEW

File: src/main/java/org/jitsi/jigasi/TranscriptionGateway.java [hunk 1]
```diff
@@ -106,15 +106,17 @@ private String getCustomTranscriptionServiceClass(String tenant)
                         REMOTE_TRANSCRIPTION_CONFIG_URL,
                         null);
 
-        if (remoteTranscriptionConfigUrl != null)
+        if (remoteTranscriptionConfigUrl != null && tenant != null)
         {
+            logger.info("Retrieving transcriber from remote URL " + remoteTranscriptionConfigUrl + "/" + tenant);
```
Comment (damencho) [inline src/main/java/org/jitsi/jigasi/TranscriptionGateway.java:111]: Do we need that log on every conference?

Comment (rpurdel) [reply src/main/java/org/jitsi/jigasi/TranscriptionGateway.java]: moved to debug

File: src/main/java/org/jitsi/jigasi/TranscriptionGateway.java [hunk 2]
```diff
@@ -133,24 +140,36 @@ private String getCustomTranscriptionServiceClass(String tenant)
...
-        return JigasiBundleActivator.getConfigurationService()
+        logger.info("Attempting to retrieve transcriber from local configuration.");
+        String customTranscriptionServiceClass
+                = JigasiBundleActivator.getConfigurationService()
                 .getString(
                         CUSTOM_TRANSCRIPTION_SERVICE_PROP,
                         null);
+        if (customTranscriptionServiceClass != null)
+        {
+            logger.info("Using " + customTranscriptionServiceClass + " as the transcriber class.");
```
Comment (damencho) [inline src/main/java/org/jitsi/jigasi/TranscriptionGateway.java:166]: Maybe leave only the print if there is a tenant enabling non default transcription. 
And please add the callContext to the log as the rest of the code.

File: src/main/java/org/jitsi/jigasi/TranscriptionGateway.java [hunk 3]
```diff
@@ -133,24 +140,36 @@ private String getCustomTranscriptionServiceClass(String tenant)
...
+        }
+        else
+        {
+            logger.info("No custom transcriber class found, falling back to default.");
```
Comment (damencho) [inline src/main/java/org/jitsi/jigasi/TranscriptionGateway.java:170]: We do not need this log.

**Review by damencho — APPROVED** [review]

**Pull request merged**